chkconfig.spec

```diff
@@ -81,9 +81,7 @@ mkdir -p $RPM_BUILD_ROOT/etc/chkconfig.d
 %{_sysconfdir}/chkconfig.d
 %{_sysconfdir}/init.d
 %{_sysconfdir}/rc.d
-%{_sysconfdir}/rc.d/init.d
```

Comment:
I don't understand this. If you also need some changes in chkconfig packaging, please create a separate PR.

Reply:
Builds won't pass otherwise.  OK to break it out into a separate commit?

Reply:
We should figure out what is wrong here, /etc/rc.d/init.d/ is the proper dir for initscripts.

Reply:
We already `%dir /etc/rc.d` so it complains about `init.d` being listed redundantly